numpy/core/_methods.py

```diff
@@ -7,6 +7,7 @@
 
 import warnings
 
+import numpy as np
```

Comment:
This looks out of place with the rest of the imports, but I am not a regular numpy contributor.

e.g. My guess is for ``np.sort`` you'd need to do something like ``from numpy.core import sort as nsort`` (to avoid clashing with the builtin sort).